### ISSUE TITLE
Ask about accuracy

### ISSUE THREAD

**Issue by TruongQToan** [zakizhou/CAPTCHA#6]
I have cloned your code and run in my computer. The loss decreased but the validation accuracy is around 11% and didn't increase. I don't understand why and am very confused. Do you have any insight into the problem? Thank you very much.

**Comment by hzy46** [comment]
have you fixed it? i have the same issue.

**Comment by TruongQToan** [comment]
I couldn't fix it so I tried this code [https://github.com/matthewearl/deep-anpr](url) instead.

**Comment by parikshit223933** [comment]
Even when the code is running fine, i am not getting an accuracy of 70%.
![image](https://user-images.githubusercontent.com/47470038/73584406-31e83d00-44be-11ea-8dcb-e416c1941d80.png)
I have done everything correctly, but still, not gained more than 14%.
trained on gpu (NVIDIA RTX 2070).
